openpype/hosts/nuke/api/workio.py

```diff
@@ -25,6 +26,13 @@ def open_file(filepath):
     # To remain in the same window, we have to clear the script and read
     # in the contents of the workfile.
     nuke.scriptClear()
+    headless = QtWidgets.QApplication.instance() is None
```

Comment:
This is quite interesting approach. Definitelly useful! Could we actually abstract this to api/lib.py as something like `is_headless` function?

Reply:
I've added it in the latest commit.